modules/regional/variables.tf

```diff
@@ -0,0 +1,32 @@
+variable "autospotting_lambda_arn" {}
+
+# Label configuration
+variable "label_context" {
+  description = "Used to pass in label module context"
+  type        = object({
+    namespace           = string
+    environment         = string
+    stage               = string
+    name                = string
+    enabled             = bool
+    delimiter           = string
+    attributes          = list(string)
+    label_order         = list(string)
+    tags                = map(string)
+    additional_tag_map  = map(string)
+    regex_replace_chars = string
+  })
+  default     = {
+    namespace           = ""
+    environment         = ""
+    stage               = ""
+    name                = ""
```

Comment:
Maybe set some of these defaults to mention AutoSpotting?

Reply:
As above followed the example in the other module. All of these will be set by the main Autospotting module.

Reply:
Cool, thanks!